From f29a77c154ac2b6736e816516a7890d5ec748c65 Mon Sep 17 00:00:00 2001
From: Karl Dietz <dekarl@mythtv.org>
Date: Sat, 8 Mar 2014 13:03:24 +0100
Subject: [PATCH] let enabled be a number (cppcheck)

Its used as a number im other places, so let it be a number over here, too.
---
 mythplugins/mythzoneminder/mythzoneminder/zmconsole.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mythplugins/mythzoneminder/mythzoneminder/zmconsole.cpp b/mythplugins/mythzoneminder/mythzoneminder/zmconsole.cpp
index e8f3c6c..f23d932 100644
--- a/mythplugins/mythzoneminder/mythzoneminder/zmconsole.cpp
+++ b/mythplugins/mythzoneminder/mythzoneminder/zmconsole.cpp
@@ -86,7 +86,7 @@ bool FunctionDialog::Create()
 void FunctionDialog::setMonitorFunction(void)
 {
     QString function = m_functionList->GetValue();
-    bool enabled = (m_enabledCheck->GetCheckState() == MythUIStateType::Full);
+    int enabled = (m_enabledCheck->GetCheckState() == MythUIStateType::Full);
 
     LOG(VB_GENERAL, LOG_INFO,
         "Monitor id : " + QString::number(m_monitor->id) +
@@ -94,7 +94,7 @@ void FunctionDialog::setMonitorFunction(void)
         ", enable value " + QString::number(m_monitor->enabled) + " -> " +
         QString::number(enabled));
 
-    if (m_monitor->function == function && m_monitor->enabled == enabled)
+    if ((m_monitor->function == function) && (m_monitor->enabled == enabled))
     {
         LOG(VB_GENERAL, LOG_WARNING,
             "Monitor Function/Enable values not changed so not updating.");
-- 
1.7.10.2

